app/models/nextUpdates/NextUpdatesResponseModel.scala

```diff
@@ -72,9 +72,8 @@ case class NextUpdateModel(start: LocalDate,
                            dateReceived: Option[LocalDate],
                            periodKey: String) extends NextUpdatesResponseModel {
 
-  def currentTime(): LocalDate = LocalDate.now()
-
-  def getNextUpdateStatus: NextUpdateStatus = if (!currentTime().isAfter(due)) Open(due) else Overdue(due)
+  def currentTime(implicit dateService: DateService): LocalDate = dateService.getCurrentDate
+  def getNextUpdateStatus(implicit dateService: DateService): NextUpdateStatus = if (!currentTime(dateService).isAfter(due)) Open(due) else Overdue(due)
```

Comment:
this method is only used in the scope of unit tests, so we can move this logic in the helpers logic in the scope of unit tests;